main.tf

```diff
@@ -54,7 +61,7 @@ resource "cloudfoundry_app" "egress_app" {
         configuration = values(local.configuration)
       })
     },
-    local.environment_passwords
+    local.environment_credentials...
```

Comment:
What's the `...` notation do? I'm having trouble googling it.

Reply:
err - sorry, I think `spread` operator is the correct term for terraform

Reply:
https://developer.hashicorp.com/terraform/language/expressions/function-calls#expanding-function-arguments

Reply:
Cool! I also found [a different, poorly documented use of `...`](https://gist.github.com/reegnz/7de97f4dd70da6320e6bf86eb883cfdd) that could definitely be useful in other contexts!